plugins/env/lib/samson_env/samson_plugin.rb

```diff
@@ -14,23 +14,23 @@ def write_env_files(dir, project, deploy_groups)
 
     def env_groups(project, deploy_groups, **kwargs)
       groups =
-        if deploy_groups.any?
-          deploy_groups.map do |deploy_group|
-            [
-              ".#{deploy_group.name.parameterize}",
-              EnvironmentVariable.env(project, deploy_group, **kwargs)
-            ]
+          if deploy_groups.any?
+            deploy_groups.map do |deploy_group|
+              [
+                  ".#{deploy_group.name.parameterize}",
+                  EnvironmentVariable.env(project, deploy_group, **kwargs)
+              ]
+            end
+          else
+            [["", EnvironmentVariable.env(project, nil, **kwargs)]]
           end
-        else
-          [["", EnvironmentVariable.env(project, nil, **kwargs)]]
-        end
-      return groups if groups.any? { |_, data| data.present? }
+      return groups if groups.any? {|_, data| data.present?}
```

Comment:
Should have spaces for blocks e.g. `{ |...`

Reply:
Fixed ... @grosser set my rubymine formatting settings,  they now follow the Zendesk convention.